src/helpers/interfaces.tsx

```diff
@@ -0,0 +1,18 @@
+export interface Posts {
+  userId: number;
+  id: number;
+  title: string;
+  body: string;
+  createdAt: any,
+  updatedAt: any,
+}
+
+export interface Comments {
```

Comment:
```suggestion
export interface Comment {
```
this is single comment